build/drand.go

```diff
@@ -44,6 +46,21 @@ var DrandConfigs = map[DrandEnum]dtypes.DrandConfig{
 		},
 		ChainInfoJSON: `{"public_key":"868f005eb8e6e4ca0a47c8a77ceaa5309a47978a7c71bc5cce96366b5d7a569937c529eeda66c7293784a9402801af31","period":30,"genesis_time":1595431050,"hash":"8990e7a9aaed2ffed73dbd7092123d6f289930540d7651336225dc172e51b2ce","groupHash":"176f93498eac9ca337150b46d21dd58673ea4e3581185f869672e59fa4cb390a"}`,
 	},
+	DrandQuicknet: {
+		Servers: []string{
+			"https://api.drand.sh",
```

Comment:
How are the same servers used for different drand networks?

Reply:
I assume it has to do with the info in `ChainInfoJSON`, but I'll let @CluEleSsUK @AnomalRoil confirm that.

Reply:
Precisely